bin/bulma-social.css

```diff
@@ -71,6 +71,72 @@ fieldset[disabled] .is-adn.focus {
   background-color: #fff;
 }
 
+
+.is-apple {
+  color: #fff;
+  background-color: #444;
```

Comment:
Change this line to ``` background-color: #000 ```

Reply:
Hi Aldi, I fixed the colors to the correct and added the the is-apple class to the main page.